[MASTER] Top border for stat bars

crkellen opened the pull request:
See #113.

WALLOFJUSTICE commented on src/interface/drawstatus.cpp:
```diff
@@ -430,35 +430,52 @@ void drawStatus()
+	// Display border between actual Health bar and "HP"
+	pos.x = 76;
+	pos.w = 38;
+	pos.h = 1;
```
Change height from 1 to 0 - Makes the border 2px high instead of 3px. Matches the 2px border for the rest of the bar.

WALLOFJUSTICE commented on src/interface/drawstatus.cpp:
```diff
@@ -480,18 +497,33 @@ void drawStatus()
+	// Display border between actual Magic bar and "MP"
+	pos.x = 12;
+	pos.w = 39;
+	pos.h = 1;
```
Change height from 1 to 0 - Makes the border 2px high instead of 3px. Matches the 2px border for the rest of the bar.

crkellen commented:
I will handle this today.
null

crkellen commented:
In addition to correcting the box heights, I realized that the HP bars are actually not centered in the X or Y axis. I corrected this, and also added a small amount of green value to the MP bar to make it significantly less jarring to look at. Finally, I increased the faint background bars' color from 16 -> 48

WALLOFJUSTICE commented:
Fine with me!

addictgamer commented:
Thanks
